.doc_gen/metadata/comprehend_metadata.yaml

```diff
@@ -131,6 +131,15 @@ comprehend_Usage_ComprehendClassifier:
   synopsis: train a custom &CMP; classifier and classify documents.
   category: Usage
   languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/comprehend
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - comprehend.java2.classifier.main
```

Comment:
I think this one's misplaced--put it under `comprehend_CreateDocumentClassifier` instead.